src/reactviews/pages/ExecutionPlan/highlightExpensiveOperations.tsx

```diff
@@ -40,13 +40,19 @@ interface HighlightExpensiveOperationsProps {
     executionPlanView: any;
     setExecutionPlanView: any;
     setHighlightOpsClicked: any;
+    context?: any;

```

Comment:
Can this be something better than `any`? (same for the others)

Reply:
I updated the context and executionPlanView type. the others are different variations of React.Dispatch<React.SetStateAction<type>, so I wasn't sure whether to update those